src/go/rdctl/cmd/reset.go

```diff
@@ -0,0 +1,164 @@
+/*
+Copyright © 2025 SUSE LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/spf13/cobra"
+
+	"github.com/rancher-sandbox/rancher-desktop/src/go/rdctl/pkg/client"
+	"github.com/rancher-sandbox/rancher-desktop/src/go/rdctl/pkg/config"
+	"github.com/rancher-sandbox/rancher-desktop/src/go/rdctl/pkg/factoryreset"
+	"github.com/rancher-sandbox/rancher-desktop/src/go/rdctl/pkg/paths"
+	"github.com/rancher-sandbox/rancher-desktop/src/go/rdctl/pkg/shutdown"
+)
+
+var (
+	vmReset      bool
+	k8sReset     bool
+	cacheReset   bool
+	factoryReset bool
+)
+
+var resetCmd = &cobra.Command{
+	Use:   "reset",
+	Short: "Reset Rancher Desktop",
+	Long: `Reset Rancher Desktop with various options:
+* --vm: Delete VM and create a new one with current settings
+* --k8s: Delete deployed Kubernetes workloads
+* --cache: Delete cached Kubernetes images
+* --factory: Delete VM and show first-run dialog on next start
+
+Options can be combined. Some combinations are redundant:
+* --factory includes --vm and --k8s (but not --cache)
+* --vm includes --k8s
+
+At least one option must be specified.`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if err := cobra.NoArgs(cmd, args); err != nil {
+			return err
+		}
+		cmd.SilenceUsage = true
+
+		// Check if any options are specified
+		if !vmReset && !k8sReset && !cacheReset && !factoryReset {
+			return fmt.Errorf("no reset options specified. Use --help to see available options")
+		}
+
+		// Handle factory reset (includes VM and K8s reset)
```

Comment:
```suggestion
		// Handle factory reset (includes VM and K8s reset, possibly cache)
```
It was initially not obvious that cache was handled here too.